remix/app/routes/auth.sign-out.test.tsx

```diff
@@ -0,0 +1,79 @@
+/**
+ * @jest-environment node
+ */
+
+/* eslint-disable @typescript-eslint/no-explicit-any */
+
+import { createCookieSessionStorage } from '@remix-run/node';
+import { Authenticator } from 'remix-auth';
+
+import { action } from './auth.sign-out';
+
+describe('Auth sign-out', () => {
+  const sessionStorage = createCookieSessionStorage({
+    cookie: { secrets: ['s3cr3t'] },
```

Comment:
Shouldn't the 's3cr3t' be in .env instead? 👀

Reply:
For this one and the env in remix_test.yml, For the reason doing this is because my testing on Github CI can't read the .env file or env variable.

To solve this I might need to get the admin permission on the repo and create the create variable [Reference](https://docs.github.com/en/actions/learn-github-actions/variables#creating-configuration-variables-for-a-repository), or is there any other way.

Reply:
Which are the tests that need .env variable?

Can you mock these tests so that they do not need the `.env`?  👀 I think mocking will be a better practice in this case.

Reply:
In this case, you can see the `action` function in the `routes/auth.sign-out.tsx` and it use a authenticator, so that authenticator need .env to work.

So here I should just mock the `action` right ?

Reply:
I see.

I only unit test in the Next.js v13 application instead of integration test like you, but for this, I remember Olivier mocked the config file [like this](https://github.com/nimblehq/js-framework-benchmark/blob/develop/nextjs-12/src/lib/middleware/passport.middleware.ts#L49) in the Next.js v12 application.

Maybe you can take a look at this and then you can ask for more advices from @olivierobert 😄. I don't have much experience with testing on the Frontend too 🥲

Reply:
Ok 😅, I will take a look at it. Thank you for the suggestion 🙏🏼.

Reply:
Hi @tednguyendev, I think for now I just do it this way [ae0d6f5](https://github.com/nimblehq/js-framework-benchmark/pull/105/commits/ae0d6f5d6e7347cdc2e4a0c13fff8feaf492a24f). Not sure is it correct or fine but just to avoid showing the env stuff in github CI file 🙉.

Reply:
Oke! LGTM! 😆